### PR TITLE
Add dark mode on HTML test report

### PR DESCRIPTION
**Pull request by philippefichet**
Issue: #12103

Signed-off-by: Philippe FICHET <philippe.fichet@laposte.net>

### Context
Add dark mode for HTML test report based on media queries level 5 and prefers-color-scheme

### Contributor Checklist
- [x] [Review Contribution Guidelines](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md)
- [x] Make sure that all commits are [signed off](https://git-scm.com/docs/git-commit#git-commit---signoff) to indicate that you agree to the terms of [Developer Certificate of Origin](https://developercertificate.org/).
- [x] Check ["Allow edit from maintainers" option](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) in pull request so that additional changes can be pushed by Gradle team
- [-] Provide integration tests (under `<subproject>/src/integTest`) to verify changes from a user perspective
- [ ] Provide unit tests (under `<subproject>/src/test`) to verify logic
- [ ] Update User Guide, DSL Reference, and Javadoc for public-facing changes
- [x] Ensure that tests pass locally: `./gradlew <changed-subproject>:check`

### Gradle Core Team Checklist
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation
- [ ] Recognize contributor in release notes

### CODE REVIEW

**Comment by blindpirate** [comment]
This is how it looks like on my machine with dark mode:

image

@philippefichet is this correct?

File: subprojects/testing-base/src/main/resources/org/gradle/api/internal/tasks/testing/report/style.css [hunk 1]
```diff
@@ -82,3 +82,31 @@ ul.linkList li {
...
+
+/* Dark mode */
+@media (prefers-color-scheme: dark) {
+    .breadcrumbs, .breadcrumbs a {
+        color: #9b9b9b;
+    }
+    #successRate, .summaryGroup {
+        border: solid 2px #d0d0d0;
+    }
+
```
Comment (rlucha) [inline subprojects/testing-base/src/main/resources/org/gradle/api/internal/tasks/testing/report/style.css]: Unneeded empty line

**Comment by blindpirate** [comment]
Thanks @philippefichet @rlucha !

**Comment by soarez** [comment]
@big-guy why was this reverted ?

**Pull request merged**